src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -142,7 +142,7 @@ else if (currentControlMode == ControlMethod.HOLD) {
     speedOnTarget = (shooter.speedOnTarget(shooterTargetSpeed, 8) && currentControlMode == ControlMethod.HOLD) || Robot.time() - startTime * 1.0e-9 > 3.5; //TODO: May need to adjust acceptable error
     hoodOnTarget = Robot.time() - startTime * 1.0e-9 > 0.75;//shooter.hoodOnTarget(shooter.calculateShooterHood(distance));
 
-    if (speedOnTarget && hoodOnTarget)
+    if (speedOnTarget && hoodOnTarget && !carousel.backwards)
```

Comment:
Add a TODO here that when Carousel is updated, this will have to change.

Reply:
you mean the shooter?

Reply:
Yes, he means the ShooterCommand.